src/instant/instantTransactions.ts

```diff
@@ -226,3 +256,19 @@ export function tradePayWithERC20(
     map(progressChange)
   );
 }
+
+const extractTradeSummary = (logs: any): { sold: BigNumber, bought: BigNumber } => {
+  let sold = new BigNumber(0);
+  let bought = new BigNumber(0);
+  logs.map((log: any) => {
+    // This is the topic for LogTrade log emitted from Simple Market
+    if (log.topics[0] === '0x819e390338feffe95e2de57172d6faf337853dfd15c7a09a32d76f7fd2443875') {
+      const tradeData = log.data.replace('0x', '');
```

Comment:
In web3.js 1.0 it is possible to call web3.eth.abi.decodeLog. Utill we migrate let's use https://github.com/ConsenSys/abi-decoder

Reply:
Yeah  I"m aware of the decodeLog method  the thing is that I don't want to get yet another library just for something so small. It's quite obvious.